stock_eggs/source-engine/egg-counter--strike--global-offensive.json

```diff
@@ -49,6 +54,96 @@
             "user_viewable": false,
             "user_editable": false,
             "rules": "required|string|max:20"
+        },
+        {
+            "name": "Ticrate",
+            "description": "Set server ticrate to 64 or 128",
+            "env_variable": "TICRATE",
+            "default_value": "128",
+            "user_viewable": true,
+            "user_editable": false,
+            "rules": "required|integer|in:64,128"
+        },
+        {
+            "name": "Player Slots",
```

Comment:
I don't support limiting player slots in any way. If a company wants to add a limit they can.